mpm/python/usrp_mpm/periph_manager/e320_periphs.py

```diff
@@ -278,12 +279,23 @@ def enable_gps(self, enable):
         self.log.trace("{} power to GPS".format(
             "Enabling" if enable else "Disabling"
         ))
-        mask = 0xFFFFFFFF ^ (0b1 << self.MB_GPS_CTRL_PWR_EN)
+        pwr_en_mask = (0b1 << self.MB_GPS_CTRL_PWR_EN)
+        rstn_mask = (0b1 << self.MB_GPS_CTRL_RST_N)
+        enabled_mask = pwr_en_mask | rstn_mask
```

Comment:
@draeman-synoptic Actually, this flips the logic of INITSURV_N (bit 2). Before, it would be HIGH, now its LOW. Was that intentional?

Reply:
You're right, I'm wrong: I thought we were setting INITSURV_N to high. But we don't, and this is no change in behaviour. I think a comment regarding this pin is in order, but I can add that.